prow/prowjobs/GoogleCloudPlatform/esp-v2/esp-v2.yaml

```diff
@@ -25,6 +27,8 @@ presubmits:
     cluster: espv2
     always_run: true
     decorate: true
+    annotations:
+      testgrid-dashboards: googleoss-test-infra
```

Comment:
I think these jobs should go on the new `googleoss-esp-v2` dashboard you created, right? They're in the espv2 cluster.

Reply:
Oops, good catch